common/collection.go

```diff
@@ -37,7 +37,7 @@ func (c *Collection) UnmarshalJSON(b []byte) error {
 
 	// Swordfish has them at the root
 	if len(c.ItemLinks) == 0 &&
-		(t.Count > 0 || t.ODataCount > 0) {
+		(t.Count > 0 || t.ODataCount > 0 || len(t.Members) > 0) {
```

Comment:
necessary because you can lose links:
**{
    "@odata.id": "/redfish/v1/Systems/system/PCIeDevices/S0B101D0/PCIeFunctions",
    "@odata.type": "#PCIeFunctionCollection.PCIeFunctionCollection",
    "Description": "Collection of PCIe Functions for PCIe Device S0B101D0",
    "Members": [
        {
            "@odata.id": "/redfish/v1/Systems/system/PCIeDevices/S0B101D0/PCIeFunctions/0"
        },
        {
            "@odata.id": "/redfish/v1/Systems/system/PCIeDevices/S0B101D0/PCIeFunctions/1"
        }
    ],
    "Name": "PCIe Function Collection",
    "PCIeFunctions@odata.count": 2
}**